packages/workflows/__tests__/__system__/cli/list/listActiveWorkflowDetails/cli.workflows.list.active.details.system.test.ts

```diff
@@ -0,0 +1,128 @@
+            tempProfileTypes: ["zosmf"],
+            testName: "create_workflow_cli"
+        });
+        systemProps = new TestProperties(testEnvironment.systemTestProperties);
+        defaultSystem = systemProps.getDefaultSystem();
+        system = testEnvironment.systemTestProperties.workflows.system;
+        owner = defaultSystem.zosmf.user;
+        wfName = `${getUniqueDatasetName(owner)}`;
+        definitionFile = `${defaultSystem.unix.testdir}/${getUniqueDatasetName(owner)}.xml`;
+        fakeDefFile = definitionFile + "FAKEFILE";
+
+        REAL_SESSION = TestEnvironment.createZosmfSession(testEnvironment);
+    });
+
+    afterAll(async () => {
+        await TestEnvironment.cleanUp(testEnvironment);
+    });
+    describe("Create workflow using uss file", () => {
+        beforeAll(async () => {
+            let error;
+            let response;
+
+            // Upload files only for successful scenarios
+            try {
+            await Upload.fileToUSSFile(REAL_SESSION, workflow, definitionFile, true);
+            } catch (err) {
+                error = err;
+            }
+
+            // Create a workflow instance in zOS/MF to list
+            response = await CreateWorkflow.createWorkflow(REAL_SESSION, wfName, definitionFile, system, owner);
+            wfKey = response.workflowKey;
+        });
+        afterAll(async () => {
+            let error;
+
+            const endpoint: string = ZosFilesConstants.RESOURCE + ZosFilesConstants.RES_USS_FILES;
+            // deleting uploaded workflow file
+            try {
+                const wfEndpoint = endpoint + definitionFile;
+                await ZosmfRestClient.deleteExpectString(REAL_SESSION, wfEndpoint);
+            } catch (err) {
+                error = err;
+            }
+
+            // deleting wf instance
+            const response: any =  await ZosmfRestClient.getExpectJSON(REAL_SESSION, "/zosmf/workflow/rest/1.0/workflows?workflowName=" + wfName);
+            response.workflows.forEach(async (element: any) => {
+                if(element.workflowName===wfName){
+                    wfKey = element.workflowKey;
+                    try {
+                        await DeleteWorkflow.deleteWorkflow(REAL_SESSION, wfKey);
+                    } catch (err) {
+                        error = err;
+                    }
+                }
+            });
+        });
+        describe("Success Scenarios", () => {
+            it("Should list active workflow details.", async () => {
+                const response = runCliScript(__dirname + "/__scripts__/command/list_active_workflow_details.sh",
+                testEnvironment, [wfKey]);
+                expect(response.stderr.toString()).toBe("");
+                expect(response.status).toBe(0);
+                expect(response.stdout.toString()).toContain("Workflow Details");
+            });
+        });
+        describe("Failure Scenarios", () => {
```

Comment:
Since these tests can be run offline, I would suggest moving them along with the help text test to an integration test file so that they can be run automatically on Jenkins.